connaisseur/flask_server.py

```diff
@@ -60,8 +60,7 @@ def mutate():
         else:
             err_log = str(traceback.format_exc())
             msg = "unknown error. please check the logs."
-        if call_alerting_on_request(admission_request, admitted=False):
-            send_alerts(admission_request, admitted=False, reason=msg)
+        AlertingConfiguration().send_alerts(admission_request, False, msg)
```

Comment:
This reads weirdly. A configuration should not send alerts. Either the Config should be named differently or it has too many capabilities